event/doc.go

```diff
@@ -0,0 +1,11 @@
+// Package event contains the abstractions for a local event bus, along with the standard events
+// that libp2p subsystems may emit.
+//
+// Source code is arranged as follows:
+// 	* doc.go: this file.
```

Comment:
```suggestion
// * doc.go: this file.
```

Reply:
I want this to be a code block, otherwise the "list" formatting will get messed up.